workflows/diagnostics/fv3net/diagnostics/offline/views/create_report.py

```diff
@@ -260,7 +260,7 @@ def render_index(config, metrics, ds_diags, ds_transect, output_dir) -> str:
     vars_3d = [v for v in ds_diags if is_3d(ds_diags[v])]
     ds_profiles = get_plot_dataset(
         ds_diags[vars_3d],
-        var_filter="time_domain_mean",
+        var_filter="time_domain_mean_model_level",
```

Comment:
technically a breaking-ish change in how the report is created, but since we rarely run the report without the compute diagnostics step anyway it seems like this would be fine